pkg/dockercreds/parse_annoted_secrets_test.go

```diff
@@ -0,0 +1,66 @@
+package dockercreds
```

Comment:
Are we testing anything internal in this package? if not can we change the package to `dockercreds_test`?

Reply:
Yep let's move this to dockercreds_test package.